packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -114,6 +115,134 @@ describe("Garbage Collection Tests", () => {
         sessionStorageConfigProvider.value.getRawConfig = oldRawConfig;
     });
 
+    describe("Configuration", () => {
+        const createGcWithPrivateMembers = (gcMetadata?: IGCMetadata, gcOptions?: IGCRuntimeOptions, snapshotCacheExpiryMs?: number) => {
+            const metadata: IContainerRuntimeMetadata | undefined = gcMetadata && { summaryFormatVersion: 1, message: undefined, ...gcMetadata };
+            const gcWithPrivates: IGarbageCollector & {
+                readonly gcEnabled: boolean;
+                readonly sweepEnabled: boolean;
+                readonly shouldRunGC: boolean;
+                readonly shouldRunSweep: boolean;
+                readonly trackGCState: boolean;
+                readonly testMode: boolean;
+                readonly latestSummaryGCVersion: GCVersion;
+                readonly sessionExpiryTimeoutMs: number | undefined;
+                readonly inactiveTimeoutMs: number;
+                readonly sweepTimeoutMs: number | undefined;
+            } = createGarbageCollector({ metadata, gcOptions, snapshotCacheExpiryMs }) as any;
+            return gcWithPrivates;
+        };
+        describe("Existing container", () => {
+            it("No metadata", () => {
+                const gc = createGcWithPrivateMembers({});
+                assert(!gc.gcEnabled, "gcEnabled incorrect");
```

Comment:
For some reason, I thought we enabled GC by default

Reply:
https://github.com/microsoft/FluidFramework/pull/10433
When did this change back to defaulting to false?

Reply:
Oh right, this is an existing container. LGTM